config.md

```diff
@@ -463,6 +449,20 @@ Here is a full example `config.json` for reference.
                 "gid": 0
             }
         ],
+        "uidMappings": [
+            {
+                "hostID": 1000,
+                "containerID": 0,
+                "size": 32000
```

Comment:
32000 looks like a non sense magic number, 32768 might be better, what do you think?